prover/prover_cli/Cargo.toml

```diff
@@ -10,11 +10,23 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+env_logger = "0.10"
+log = "0.4"
```

Comment:
1. Is there a reason not to use workspace deps here?
2. Why do we need `env_logger` and `log` when we use `tracing` and `vlog` (or `tracing-subscriber`, if you have to) now?